ee/cis/macos-13/cis-policy-queries.yml

```diff
@@ -791,6 +791,38 @@ spec:
 ---
 apiVersion: v1
 kind: policy
+spec:
+  name: CIS - Ensure Security Auditing Retention Is Enabled
+  platforms: macOS
+  platform: darwin
+  description: |
+    The macOS audit capability contains important information to investigate security or operational issues. This resource is only completely useful if it is retained long enough to allow technical staff to find the root cause of anomalies in the records.
+    Retention can be set to respect both size and longevity. To retain as much as possible under a certain size, the recommendation is to use the following:
+    expire-after:60d OR 5G
+    This recomendation is based on minimum storage for review and investigation. When a third party tool is in use to allow remote logging or the store and forwarding of logs, this local storage requirement is not required.
+  resolution: |
+    Automated method:
+    Ask your system administrator to deploy the following script which will ensure proper Security Auditing Retention:
+    cp /etc/security/audit_control ./tmp.txt; origExpire=$(cat ./tmp.txt  | grep expire-after);  sed "s/${origExpire}/expire-after:60d OR 5G/" ./tmp.txt > /etc/security/audit_control; rm ./tmp.txt;
+  query: |
+    SELECT 1 WHERE
+    EXISTS ( SELECT line, 
+             CAST ( regex_match(line, 'expire-after:(\d+)d', 1) AS INTEGER ) AS val 
```

Comment:
My workstation has `expire-after:10M`, so this regex won't work.

Maybe the following (`\w*` means zero or more word characters):
`regex_match(line, '^expire-after:(\d+)\w*', 1) AS INTEGER`

Reply:
Ah, dismiss my comment, looks like we want `>=60d OR >=5G`. Gotcha.

Reply:
10M (10 Mega Bytes) is not a legit value. So it's good that it fails.

Reply:
Should be:
expire-after:60d OR 5G

60d or higher
5G or higher